### PR TITLE
Added camel-karaf-coap wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -643,22 +643,16 @@
...
-<!--        <bundle>wrap:mvn:org.eclipse.californium/californium-core/${californium-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector/${californium-version}$overwrite=merge&amp;Import-Package=net.i2p.crypto.eddsa;resolution:=optional</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector-tcp-netty/${californium-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/scandium/${californium-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-coap/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-coap' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>camel-netty</feature>-->
+        <feature>bouncycastle</feature>-->
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Should not end with `-->`

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: fixed

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -581,22 +581,16 @@
...
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector/${californium-version}$overwrite=merge&amp;Import-Package=net.i2p.crypto.eddsa;resolution:=optional</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/element-connector-tcp-netty/${californium-version}</bundle>-->
-<!--        <bundle>wrap:mvn:org.eclipse.californium/scandium/${californium-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-coap/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-coap' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>camel-netty</feature>
+        <feature>bouncycastle</feature>
+        <bundle>wrap:mvn:org.eclipse.californium/californium-core/${californium-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Why do you use the wrap protocol here? `californium-core` is an OSGI bundle so it should not be needed

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: Please also add the attribute `dependency='true'` to only install the dependency if it has not yet been installed

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**